Bottom: 26ad008e38eac029a1cbb653857a20c45e6f2d3f
Top:    b15bd12b37bacd26d3e2b83e6a7c680ba0f7d494
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-30 14:26:49 -0500

Rearrange DEBIAN_FRONTEND envar in Dockerfiles


---
diff --git a/docker/buildbot-worker-ntel/Dockerfile b/docker/buildbot-worker-ntel/Dockerfile
index bea0ea8..7965c9e 100644
--- a/docker/buildbot-worker-ntel/Dockerfile
+++ b/docker/buildbot-worker-ntel/Dockerfile
@@ -12,9 +12,9 @@ MAINTAINER  Buildbot maintainers
 WORKDIR /buildbot
 
 # Install security updates and required packages
-RUN DEBIAN_FRONTEND=noninteractive \
-    apt-get update \
-&&  apt-get -y install -q \
+RUN apt-get update \
+&&  DEBIAN_FRONTEND=noninteractive \
+    apt-get -y install -q \
        alien \
        build-essential \
        chrpath \
@@ -84,9 +84,9 @@ RUN useradd --comment "Buildbot Server" --home-dir "/home/buildbot" --shell "/bi
 &&  chown -v -R buildbot:buildbot "/buildbot" \
 &&  chown -v -R buildbot:buildbot "/home/buildbot"
 
-RUN DEBIAN_FRONTEND=noninteractive \
-    apt-get update \
-&&  apt-get -y install -q \
+RUN apt-get update \
+&&  DEBIAN_FRONTEND=noninteractive \
+    apt-get -y install -q \
        vim \
        strace \
        tmux
